contrib/test/integration/test.yml

```diff
@@ -21,5 +21,5 @@
   shell: "CGROUP_MANAGER=cgroupfs STORAGE_OPTIONS='--storage-driver=overlay{{ extra_storage_opts | default('') }}' make localintegration >& {{ artifacts }}/testout.txt"
   args:
     chdir: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-incubator/cri-o"
-  async: 5400
-  poll: 30
+  async: '{{ 2 * 60 }}'
```

Comment:
I think [we should bump this *down*][1].

[1]: https://github.com/kubernetes-incubator/cri-o/issues/1424#issuecomment-371027181

Reply:
Aye, it's just an experiment WRT the kernel bug to see if it's reproducible.

Reply:
oops, that's two minutes...not hours.

Reply:
> Aye, it's just an experiment WRT the kernel bug to see if it's reproducible.

[Looks reproducible][1] to me ;).  Something like a 75% chance of the bug?

[1]: https://ci.openshift.redhat.com/jenkins/job/test_pull_request_crio_integration_rhel/buildTimeTrend

Reply:
IKR :frowning: but if it is that specific bug, it should eventually procede...hence the extra-long timeout.  Also gives me a chance to ssh into the VM and see if the kernel is complaining.